### ISSUE TITLE
webvisum should be integrated into NVDA

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3305]
**Reported by Animesh.mr on 2013-06-27 13:48**
hi it is a new feature that i want in nvda it is only available with firefox but for other user it is not so if done that no matur what brouzer u use this feature will be there and i think this the first screen reader to implement this this may lead to good future to n v d a isn't it ?

**Comment by derekriemer** [comment]
closing as invalid as webvisum is dead and this is a paid feature (Most likely).
